ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-serialization.js

```diff
@@ -98,15 +98,15 @@ const Point = {
 
     return {
       mode: 'circle',
-      locationType: 'latlon',
+      locationType: 'dd',
       lat,
       lon,
       radius: width,
       radiusUnits: unit,
     }
   },
   'location->json': location => {
-    const { lat = 0, lon = 0, radius = 1, radiusUnits = 'meters' } = location
+    const { lat = '', lon = '', radius = 0, radiusUnits = 'meters' } = location
```

Comment:
:pencil2: verify this should be 0 and not `''`

Reply:
on the same note, `width`, above, and `lineWidth`, up in the `LineString` and `MultiLineString` definitions in this file, are currently set to 1. I'm wondering if I need to also change these.

Reply:
I changed it to `''` here and everything seems to be fine. I'm going to wait to commit this change until after investigating what i mentioned above

Reply:
it seems that, from the entry point of the anyGeo search, the `radius` will never be undefined (it's an empty string if you don't type anything in), so this default value never gets used. Not sure what we should put here, so I'll leave this open for now